site/_posts/2017-11-27-ray-0.3-release.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Ray: 0.3 Release"
+excerpt: "This post announces the release of Ray 0.3."
+date: 2017-11-29 00:00:00
+---
+
+We are pleased to announce the Ray 0.3 release. This release includes a number
+of bug fixes, as well as [**distributed actors**][1] and a [**hyperparameter
+search library**][2] (including integration with [**RLlib**][3]).
```

Comment:
I would really emphasize the main release features, particularly RLlib-related. How about : 
This release introduces **Ray.tune** --- the new RLlib-integrated hyperparameter search library, distributed actor RMI, and a number of stability and performance enhancements and bug fixes.

Reply:
I'd like to hold off on emphasizing RLlib, until we have the refactoring completed / paper posted, i.e. 0.4